ut/s3_get_object_action_test.cc

```diff
@@ -275,6 +275,12 @@ TEST_F(S3GetObjectActionTest, ValidateObjectOfSizeZero) {
       .Times(AtLeast(1));
   EXPECT_CALL(*ptr_mock_request, send_reply_end()).Times(1);
 
+  std::map<std::string, std::string> meta_map{{"key", "value"}};
```

Comment:
It possible to see here std::map<>::map(std::initializer_list<const char*>({...})).
Hope, the compiler see what you want.
Suggest to test for ensure

Reply:
yes, UTs have compiled and ran successfully. I have fired Jenkins job as well..

Can you elaborate more on what's the issue here though ?